### PR TITLE
TW-585 Move settings from tab-bar and add nft tab

### PR DESCRIPTION
**Pull request by svjatoslavtt**

### CODE REVIEW

File: src/screens/wallet/settings/settings.styles.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { StyleSheet } from 'react-native';
+
+import { formatSize } from '../../../styles/format-size';
+
+export const SettingsStyles = StyleSheet.create({
```
Comment (herkoss) [inline src/screens/wallet/settings/settings.styles.ts:5]: We use `createUseStyles` function

**Review by herkoss — DISMISSED** [review]
Good job!

**Review by Boltman92 — APPROVED** [review]

File: src/screens/wallet/settings/settings.styles.ts (hunk 1 above)
Comment (svjatoslavtt) [reply src/screens/wallet/settings/settings.styles.ts:5]: I didn't write this code, I just copied it from the previous component which I removed :)

**Review by herkoss — APPROVED** [review]

**Review by Boltman92 — APPROVED** [review]

**Review by vladvn1905 — APPROVED** [review]
build version: 1.14_589 Android done

build version: 1.14_589 ios done

**Pull request merged**